plugin/trino-sqlserver/src/main/java/io/trino/plugin/sqlserver/SqlServerClient.java

```diff
@@ -211,6 +212,8 @@ public SqlServerClient(
     {
         super(config, "\"", connectionFactory, queryBuilder, identifierMapping);
 
+        this.identifierMapping = identifierMapping;
```

Comment:
`rnn`

Reply:
sorry, don't know what's rnn mean

Reply:
requireNonNull from Guava. Checks if variable is null, returns it if it's not, throws NPE otherwise

Reply:
> sorry, don't know what's rnn mean

Sth like this `this.identifierMapping = requireNonNull(identifierMapping, "identifierMapping is null");`

If you don't want to write such code manually, we can configure a Live Template in Intellij, you can refer to https://gist.github.com/findepi/b597d2bc91e5a30c2a44fd65fe252f0b